SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Channel/Channel.cs

```diff
@@ -550,7 +550,14 @@ private void CheckPendingLogs(State state)
                     _batchScheduled = true;
                     Task.Run(async () =>
                     {
-                        await TriggerIngestionAsync(state).ConfigureAwait(false);
+                        try
+                        {
+                            await TriggerIngestionAsync(state).ConfigureAwait(false);
+                        } 
+                        catch (StatefulMutexException exc)
+                        {
+                            AppCenterLog.Warn(AppCenterLog.LogTag, "The check pending logs operation has been canceled");
```

Comment:
Is `StatefulMutexException` can only be thrown by pending logs check?

Reply:
Yes, when I investigated this issue I was able to reproduce throw this exception only here.